onedal/spmd/ensemble/__init__.py

```diff
@@ -14,6 +14,6 @@
 # limitations under the License.
 # ==============================================================================
 
-from .forest import RandomForestClassifier, RandomForestRegressor
+from onedal.ensemble import RandomForestClassifier, RandomForestRegressor
```

Comment:
```suggestion
from .ensemble import RandomForestClassifier, RandomForestRegressor
```
In general we have been using relative paths in cases like this

Reply:
My branch deletes `forest.py` and we're using the batch version directly. After removing `EstimatorBaseSPMD` there is no need to create child classes from the batch version. This change gets rid of a lot of code that doesn't really do anything and only makes it harder to understand what's happening.

https://github.com/intel/scikit-learn-intelex/blob/main/onedal/spmd/ensemble/forest.py

Reply:
Nevertheless I have added relative imports consistently (`...ensemble` in this case)